src/cli/Microsoft.DotNet.UpgradeAssistant.Cli/Commands/Upgrade/ConsoleUpgrade.cs

```diff
@@ -70,6 +81,11 @@ public async Task RunAsync(CancellationToken token)
                     step = await _upgrader.GetNextStepAsync(context, token);
                 }
 
+                if (_writerProvider.TryGetWriter("html", out var writer))
```

Comment:
Should try and mimic the code in the Analyze command where we pull out the format specified by the user, try and get a writer for that, etc.
https://github.com/dotnet/upgrade-assistant/blob/e1207a2519a12504527f0cfddd4d57e0b3553abd/src/cli/Microsoft.DotNet.UpgradeAssistant.Cli/Commands/Analyze/ConsoleAnalyze.cs#L65